appeals/e2e/cypress/e2e/back-office-appeals/reviewLpaq.spec.js

```diff
@@ -7,14 +7,14 @@ import { UpdateDueDatePage } from '../../page_objects/updateDueDatePage';
 const appealsListPage = new AppealsListPage();
 const updateDueDatePage = new UpdateDueDatePage();
 
-describe.skip('Appeals feature', () => {
+describe('Appeals feature', () => {
 	beforeEach(() => {
 		cy.login(users.appeals.caseAdmin);
 	});
 
 	it('Complete LPAQ', () => {
-		cy.visit('/appeals-service/appeals-list');
-		appealsListPage.clickAppealFromList(18);
+		cy.visit('/appeals-service/all-cases');
+		appealsListPage.clickAppealFromList(30);
```

Comment:
This approach feels fragile, as the appeal we want has already changed in position. I suggest that we revisit this and select the appeal based on data attribute, so that the position of it in the list doesn't matter

Reply:
I agree, once we have the data attribute we can refine all of our tests that use positions, element ids or text copy to make the more stable.